packages/dicomImageLoader/codecs/jpegLossless.js

```diff
@@ -0,0 +1,2061 @@
+/* eslint-disable */
```

Comment:
Ditto, please exclude codecs built files.  They can be fairly easily included as required.

Reply:
I'll do that later, there is no build for now. I need this to get merged so that the typescript gets added